firstvoices/backend/views/sites_views.py

```diff
@@ -96,3 +96,76 @@ def list(self, request, *args, **kwargs):
         ]
 
         return Response(data)
+
+
+@extend_schema_view(
+    list=extend_schema(
+        description="A list of language sites a given user is a member of, "
+        "grouped by language. If there are no accessible sites or "
+        "the user does not have membership to any site then the "
+        "list will be empty. Sites with no specified language "
+        "will be grouped under 'Other'.",
+        responses={
+            200: inline_serializer(
+                name="InlineLanguageSerializer",
+                fields={
+                    "language": serializers.CharField(),
+                    "sites": SiteSummarySerializer(many=True),
+                },
+            ),
+        },
+    ),
+)
+class MySitesViewSet(
+    AutoPermissionViewSetMixin, mixins.ListModelMixin, viewsets.GenericViewSet
+):
```

Comment:
Oh, I get it. Only adding the list mixin. That works! Ignore my comment about only registering the list view in the urls